RxSwift/Observables/Zip+arity.swift

```diff
@@ -71,7 +71,9 @@ final class ZipSink2_<E1, E2, O: ObserverType> : ZipSink<O> {
             rxFatalError("Unhandled case (Function)")
         }
 
+        #if compiler(<5.0)
```

Comment:
`Zip+arity.swift` is generated from `Zip+arity.tt`. There is a Preprocessor in `Preprocessor/Preprocessor.xcodeproj` inside of this repo :)